### DISCUSSION TITLE
Ordering of synth: How to ensure everything else is created?

### DISCUSSION THREAD

**Discussion by dkershner6** [projen/projen#3266] [Q&A] [answered]
Actual problem: I need to insert a step into every job of every workflow at the beginning of the job.  Writing the code to do so is no issue, but the Publisher jobs are not created yet, even if I use `preSynthesize`, they are undefined.  In `postSynthesize` I see them just fine, but making changes doesn't do anything.

Is there a way to get to the back of the line?

Thank you.

**Comment by mrgrain** [comment, marked as answer]
This should work: https://projen.io/docs/concepts/escape-hatches#object-file-patches
